source/geod24/concurrency.d

```diff
@@ -940,6 +940,14 @@ class FiberScheduler
     private bool terminated;
     private bool dispatching;
 
+    private Mutex fibers_lock;
```

Comment:
It should be introduced in the commit it is used by

Reply:
I combined this with the two `commit` behind.